mbf_abstract_nav/include/mbf_abstract_nav/abstract_controller_execution.h

```diff
@@ -219,6 +220,14 @@ namespace mbf_abstract_nav
      */
     void setVelocityCmd(const geometry_msgs::TwistStamped &vel_cmd_stamped);
 
+    /**
+     * @brief Check if the robot's actual velocity is ignoring the cmd_vel
+     * @param robot_stopped true if robot is stopped, false otherwise
+     * @param cmd_velocity the commanded velocity being published to the robot
+     * @return true if there the ignoring time exceeded tolerance, false otherwise
+     */
+    bool checkVelocityIgnore(const bool& robot_stopped, const geometry_msgs::TwistStamped& cmd_velocity);
```

Comment:
```suggestion
    bool checkVelocityIgnore(const geometry_msgs::Twist& cmd_vel);
```
S1: Check if the robot is stopped inside the function rather than passing the bool from outside
S2: Pass the twist without the stamp since you're not using it